core/src/main/res/values/arrays.xml

```diff
@@ -179,12 +179,14 @@
         <item>@string/drawer_feed_counter_unplayed</item>
         <item>@string/drawer_feed_counter_downloaded</item>
         <item>@string/drawer_feed_counter_none</item>
+        <item>@string/drawer_feed_counter_downloaded_unplayed</item>
```

Comment:
I think "none" should stay the last option, because it is fundamentally different from the others. Could you please re-order the two arrays, so that it is the second-last item?

Reply:
Done